docs/expectation_gallery/azure-pipelines-expectation-gallery.yml

```diff
@@ -15,35 +15,35 @@ schedules:
     - develop
   always: false # Will only trigger if the state of the codebase has changed sinced the last scheduled run
 
-resources:
-  containers:
-  - container: postgres
-    image: postgres:11
-    ports:
-    - 5432:5432
-    env:
-      POSTGRES_DB: "test_ci"
-      POSTGRES_HOST_AUTH_METHOD: "trust"
-  - container: mysql
-    image: mysql:8.0.20
-    ports:
-      - 3306:3306
-    env:
-      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
-      MYSQL_DATABASE: test_ci
-  - container: mssql
-    image: mcr.microsoft.com/mssql/server:2019-latest
-    env:
-      ACCEPT_EULA: Y
-      MSSQL_SA_PASSWORD: ReallyStrongPwd1234%^&*
-      MSSQL_DB: test_ci
-      MSSQL_PID: Developer
-    ports:
-      - 1433:1433
-  - container: trino
-    image: trinodb/trino:400
-    ports:
-      - 8088:8080
+#resources:
```

Comment:
We do want these resources when we start building the gallery. I commented them out for now so it would be easier to manually add them back in. I did this because I don't think we want to just revert this PR.

Reply:
Just curious: does it negatively impact us to still have the resources, so long as the jobs don't run?

Reply:
Could we leave a comment in this file as to why we're commenting it out?

Reply:
I've left a comment.

Anecdotally the pipeline takes longer to spin up, but I haven't really measured this. I'm not sure if it uses workers but I suspect it does.